setup.py

```diff
@@ -0,0 +1,13 @@
+"""Setuptools package definition."""
+
+from setuptools import find_packages, setup
+
+setup(
+    name='caluma',
+    version='0.0.0',
+    author='Adfinis SyGroup AG',
+    author_email='https://adfinis-sygroup.ch/',
```

Comment:
Are we sure that "Adfinis SyGroup AG" should be the author? Also, this email is not an email address..

Reply:
Removed author and author_emai. Not really relevant right now anyway.